Build/Run-Tests.ps1

```diff
@@ -12,8 +12,10 @@ $opencover_console = "packages\OpenCover.$opencover_version\tools\OpenCover.Cons
     -excludebyfile:*\*Designer.cs `
     -output:"OpenCover.GitExtensions.xml" `
     -target:"nunit3-console.exe" `
-    -targetargs:"$testAssemblies"
-if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode) }
+    -targetargs:"$testAssemblies --workers=1 --timeout=90000"
```

Comment:
How does this affect a duration of the build?

Reply:
It does. It takes less time to execute the tests. I guess the single CPU core (with hyperthreading) works more effective with less threads. Anyway the tests from the different test assemblies are still run in parallel.